media/doc/WINESYNC.txt

```diff
@@ -263,9 +263,9 @@ check Wine current sources first as it may already be fixed.
 
 sdk/lib/3rdparty/strmbase           # Synced to WineStaging-3.3
 
-sdk/lib/rtl/actctx.c                # Partly synced with WineStaging-1.9.16
-sdk/lib/rtl/timerqueue.c            # Partly synced with WineStaging-1.7.55
-sdk/lib/rtl/wait.c                  # Partly synced with WineStaging-1.7.55
+sdk/lib/rtl/actctx.c                # Synced to wine-5.18
+sdk/lib/rtl/timerqueue.c            # Synced to wine-5.18
+sdk/lib/rtl/wait.c                  # Synced to wine-5.18
```

Comment:
Shouldn't the `wine` word start from a capital letter? Or, if it is automatically updated by our winesync.py script, then it's OK as-is (yeah, I see that other similar lines with this word are also have a `wine` instead of `Wine`).